localstack/cli/localstack.py

```diff
@@ -193,8 +193,14 @@ def cmd_stop():
     help="Block the terminal and follow the log output",
     default=False,
 )
+@click.option(
+    "--tail",
```

Comment:
could we maybe add `-n` as shorthand? this would be consistent with `tail` and also `docker logs`.

Reply:
Thank you for the suggestion. I have added the `-n` option as a shorthand for `--tail` in the latest commit. It is consistent with the `tail` option and the `docker logs` command, as you suggested.